Module seven (#8)
aliwen-soft/DevOps-Course-Starter: README.md

docker build --target production --tag todo-app:prod .
docker run -p 3000:3000 --env-file .env todo-app:prod
```

To run the tests in the docker container use:
```
docker build --target test --tag todo-app:test .
docker run --env-file .env.test todo-app:test

> ```

Reviewer: Thanks, super clear and easy

## Pipeline

> The github pipeline i set to trigger on pullrequest to main and on pushes as long as its not just documentation.

Reviewer: To quote your commit history; 
> typo lol
😄 
```suggestion
The github pipeline is set to trigger on pull requests to main and on pushes as long as its not just documentation.
```
